shoop/admin/modules/orders/json_order_creator.py

```diff
@@ -13,8 +13,8 @@
 from django.utils.translation import ugettext as _
 
 from shoop.core.models import (
-    Address, Contact, CompanyContact, PersonContact, OrderLineType,
-    OrderStatus, PaymentMethod, Product, ShippingMethod, Shop
+    Address, CompanyContact, Contact, OrderLineType, OrderStatus, PaymentMethod, PersonContact, Product, ShippingMethod,
+    Shop
 )
```

Comment:
These would be much more readable with the 79 columns. Since they will be splitted to multiple lines anyway, why make it so wide then?